memory.c

```diff
@@ -175,6 +175,26 @@ memory_get_rom_bank()
 	return rom_bank;
 }
 
+// Control the GIF recorder
+//        bit:  7     6  5  4  3  2  1           0
+// record_gif: [ctrl][0][0][0][0][0][continuous][active]
+void
+emu_recorder_set(uint8_t newstate)
+{
```

Comment:
imho it would be easier to read if the possible values(or bit mask) for record_gif would be enums or #defines instead of 0, 1, 128,255.

Reply:
Added #defines, it does make the code clearer.